partners_api/partners_api_tests/facebook_tests.cpp

```diff
@@ -0,0 +1,51 @@
+#include "testing/testing.hpp"
+
+#include "indexer/classificator.hpp"
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature_data.hpp"
+
+#include "partners_api/facebook_ads.hpp"
+
+namespace
+{
+UNIT_TEST(Facebook_GetBanner)
+{
+  try
+  {
+    classificator::Load();
+  }
+  catch (RootException const & e)
```

Comment:
Do we really need this catch? The test won't work if its not possible to load classificator, on the other case main() of tests catches all RootExceptions and prints diagnostic.